lib/plugins/config/settings/config.metadata.php

```diff
@@ -219,6 +219,8 @@
 $meta['xsendfile']   = array('multichoice','_choices' => array(0,1,2,3),'_caution' => 'warning');
 $meta['renderer_xhtml'] = array('renderer','_format' => 'xhtml','_choices' => array('xhtml'),'_caution' => 'warning');
 $meta['readdircache'] = array('numeric');
+$meta['search_limit_to_first_ns'] = array('numeric', '_min' => 0);
+$meta['search_default_fragment_behaviour'] = array('multichoice','_choices' => array('exact', 'starts_with', 'ends_with', 'contains'),);
```

Comment:
The defaults are missing in `conf/dokuwiki.php`

Reply:
Nope 🙂